tests/performance_benchmarks.py

```
@@ -0,0 +1,256 @@
+"""
+Performance benchmarking tests for Mad Spark Alt.
+
+These tests measure execution time and memory usage for key operations.
+"""
+
+import asyncio
+import time
+from memory_profiler import profile
```

Ruff (0.11.9)

`memory_profiler.profile` imported but unused

Remove unused import: `memory_profiler.profile`

(F401)